code/main/edge/konductorConfig.brs

```diff
@@ -0,0 +1,52 @@
+' ********************** Copyright 2024 Adobe. All rights reserved. **********************
+' *
+' * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+' * you may not use this file except in compliance with the License. You may obtain a copy
+' * of the License at http://www.apache.org/licenses/LICENSE-2.0
+' *
+' * Unless required by applicable law or agreed to in writing, software distributed under
+' * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+' * OF ANY KIND, either express or implied. See the License for the specific language
+' * governing permissions and limitations under the License.
+' *
+' *****************************************************************************************
+
+' ******************************* MODULE: EdgeRequestWorker *******************************
+
+function _adb_KonductorConfig() as object
+
+    return {
+        _stateStore: invalid,
+        _locationHint: invalid,
+
+        getStateStore: function() as object
+            return m._stateStore
+        end function,
+
+        getLocationHint: function() as dynamic
+            ' Extract location hint for  "scope": "EdgeNetwork"
+            return m._locationHint
```

Comment:
Is it required for Roku to handle the expiration date for the location hint?
https://github.com/adobe/aepsdk-edge-ios/blob/dev/Sources/EdgeProperties.swift#L25

Reply:
Yes we will have to honor that. There might not be instances where people would move this OTT devices but I think as per the contract we still have to honor that.

Reply:
@kevinlind I wanted to add that as a separate PR as this is already 500 LOC. I also have mores tests to add to this PR.

Reply:
I also see mobile SDK handling expiration date for state:store payloads.
https://github.com/adobe/aepsdk-edge-ios/blob/dev/Sources/EdgeNetworkHandlers/StoreResponsePayload.swift#L27

Reply:
Yes working on that part.